core/src/config.cpp

```diff
@@ -16,6 +16,7 @@ Config::Config(const RunMode &mode, const std::string &filePath)
       general_({configYaml_["general"]["fakeUrl"].as<std::string>(),
                 configYaml_["general"]["method"].as<std::string>(),
                 configYaml_["general"]["timeWait"].as<unsigned int>(),
+                configYaml_["general"]["timeout"].as<unsigned int>(),
```

Comment:
Please consider using timeout as seconds, in this case `unsigned int` is too bit